components/src/core/DropdownToolbar/DropdownToolbar.tsx

```diff
@@ -160,6 +160,12 @@ const DropdownToolbarRender: React.ForwardRefRenderFunction<unknown, DropdownToo
         [onOpen]
     );
 
+    useEffect(() => {
+        console.warn(
+            `DropdwonToolbar component has been deprecated and will be removed in version 6.0.0. Instead of DropdwonToolbar use ToolbarMenu component in regular Toolbar like Mui Appbar, BLUI Appbar.`
```

Comment:
```suggestion
            `The DropdownToolbar component has been deprecated and will be removed in version 6.0.0. Instead of DropdownToolbar, use the ToolbarMenu component inside a MUI Toolbar or Brightlayer UI AppBar component.`
```

Reply:
Agree with Huayun that it would be nice to see a link to the ToolbarMenu docs page in here as well.